cedars/app/auth.py

```diff
@@ -106,12 +129,29 @@ def register():
             flash('Registration successful.')
             if no_admin:
                 login_user(User(db.get_user(username)))
-
+            init_pines()
             return render_template('index.html', **db.get_info())
         flash(error)
     return render_template('auth/register.html', **db.get_info())
 
 
+def verify_external_token(token, project_id, user_id):
+    """Verify the bearer token with the external API"""
+    # Replace this URL with your actual external API endpoint
+    api_url = f'{os.getenv("SUPERBIO_API_URL")}/users/{user_id}/cedars_projects/{project_id}'
+    headers = {"Authorization": f"Bearer {token}"}
+    logger.info(f"Verifying token with {api_url}")
+    logger.info(f"Headers: {headers}")
+    try:
+        response = requests.get(api_url, headers=headers, verify=True)
+        if response.status_code == 200:
+            # Assuming the API returns user data on successful verification
+            return response.json()
+        return None
```

Comment:
you can replace this with

```python
try:
  ....
finally:
  return None
```

Reply:
The finally keyword does not work in this use case as the function always returns None.